docs/features/nullable-reference-types.md

```diff
@@ -22,54 +22,43 @@ namespace System.Runtime.CompilerServices
         AllowMultiple = false)]
     public sealed class NullableAttribute : Attribute
```

Comment:
>NullableAttribute [](start = 24, length = 17)

Not blocking this PR: we'll need to think about how the values of the parameters will be stored in the fields of NullableAttribute. Jared mentioned we need to store those values so that they are accessible from reflection.